checker/src/main/java/org/checkerframework/checker/signature/SignatureAnnotatedTypeFactory.java

```diff
@@ -10,10 +10,7 @@
 import java.util.Set;
 import javax.lang.model.element.AnnotationMirror;
 import javax.lang.model.element.ExecutableElement;
-import org.checkerframework.checker.signature.qual.BinaryName;
-import org.checkerframework.checker.signature.qual.InternalForm;
-import org.checkerframework.checker.signature.qual.SignatureBottom;
-import org.checkerframework.checker.signature.qual.SignatureUnknown;
+import org.checkerframework.checker.signature.qual.*;
```

Comment:
Please don't introduce wildcard imports.  They are not used elsewhere and we wish to keep the code style consistent.

(If you did want to make this change, then you should make it in a separate pull request to keep this one simple.)